scripts/download_file.py

```diff
@@ -167,7 +173,7 @@ def calculate_md5(file_path: str) -> str:
     hash_md5 = hashlib.md5()
     with open(file_path, "rb") as fh:
         chunk = fh.read(BLOCK_SIZE)
-        while chunk:
+        while chunk is not None:
```

Comment:
I did not test, but likely the same issue as above.

Reply:
Addressed.